data-types.md

```diff
@@ -122,8 +122,10 @@ A standard point of vehicle telemetry. References to latitude and longitude impl
 | `journey_id`      | UUID            | Required               | If telemetry occurred during a trip, the ID of the journey.  If not in a trip, `null`.
 | `stop_id`         | UUID            | Required if Applicable | Stop that the vehicle is currently located at. Only applicable for _docked_ Micromobility. See [Stops][stops] |
 | `location`        | [GPS][gps]      | Required               | Telemetry position data |
+| `location_type`   | Enum            | Required if Known      | If detectable and known, what type of location the device is on or in. One of `street`, `sidewalk`, `crosswalk`, `garage`, `bike_lane`.   |
 | `battery_percent` | Integer         | Required if Applicable | Percent battery charge of vehicle, expressed between 0 and 100 |
 | `fuel_percent`    | Integer         | Required if Applicable | Percent fuel in vehicle, expressed between 0 and 100 |
+| `tipped_over`     | Boolean         | Required if Known      | If detectable and known, is the device tipped over or not? Default is 'false'. |
```

Comment:
Does this look right @droverai? See #794

Reply:
Looks good. PathPilot will share the tipped_over status every second. Our current location feed to the providers will include a flag "tipped_over" with `true` or `false` value. In addition, we will be sending a one-off special event when the vehicle is tipped over. This way they have per-second status and an event as well.

Reply:
That's great. Since this field is in telemetry, it will be sent as frequently as the agency and operator agree to. Each second is not unheard of though it may be slightly less frequent depending on the vehicle state.